packages/app/components/jsonschemaform/widgets/TitlePropertyWidget.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+function TitlePropertyWidget(props) {
+    const {
+        id,
+        placeholder,
+        required,
+        disabled,
+        readonly,
+        autofocus,
+        value,
+        onChange,
+        onBlur,
+    } = props
+
+    return (
+        <input
+            type="text"
+            className="form-control"
+            id={id}
+            placeholder={placeholder}
+            required={required}
+            disabled={disabled || readonly}
+            autoFocus={autofocus || false}
+            value={value}
+            onBlur={event => {
+                onChange(event.target.value.trim())
```

Comment:
I think this all makes sense to me, but I was wondering if we should call the passed-in `onBlur` prop (probably a function reference)? [The documentation](https://react-jsonschema-form.readthedocs.io/en/v4.2.2/advanced-customization/custom-widgets-fields/#adding-your-own-custom-widgets) says that it should take in an id and value...so maybe if there's something that React JSON Schema Form expects from the `onBlur` callback, we could ensure that it gets that functionality by including it below line 27.

Reply:
Yes, I have added that before but since it was not updating value I removed it. but looks like react JSON schema form expects that. I can add this back onBlur(id, event.target.value) below line 27.